### ISSUE TITLE
[BUG] Bastille sysrc multiple space delimited entries

### ISSUE THREAD

**Issue by jaredwolff** [BastilleBSD/bastille#183]
**[MANDATORY] Describe the bug [MANDATORY]**

Adding elements into `rc.conf` using the `sysrc` command is awesome. Behavior is as expected for single elements. When inserting a much larger line of text with spaces, those get placed into their own lines. Whereas the expected behavior is to be associated in a single line. (Sorry it's a little hard to explain. See below)

**[MANDATORY] Bastille and FreeBSD version (paste ``bastille -v && freebsd-version -kru`` output)**

```
# bastille -v && freebsd-version -kru
0.6.20200202
12.1-RELEASE-p2
12.1-RELEASE-p2
12.1-RELEASE-p2
```

**[MANDATORY] How did you install bastille? (port/pkg/git)**

`pkg`

**[optional] Steps to reproduce?**

Running `bastille sysrc jailname test="ONE=yes TWO=no"` creates two entries:

```
bastille sysrc commento test="ONE=yes TWO=no"
[commento]:
test:  -> ONE=yes
TWO:  -> no
```

**[optional] Expected behavior**

Running `sysrc` itself will yield the following in `/etc/rc.conf`

```
sysrc test="ONE=yes TWO=no"
test:  -> ONE=yes TWO=no
``

My current solution is to run the command several times with the `+=` syntax. It would be great to do it in one fell swoop though!

**Comment by jpoa** [comment]
Since sysrc is being handled here I would like to point out a bug (?) with SYSRC (maybe codebase is shared).
If the template is applied multiple times things go wrong, its best explained with this output I guess:

1st time:
```
[test]:SYSRC -- START
sshd_enable: YES -> YES
[test]:SYSRC -- END
```

2nd time:
```
[test]:SYSRC -- START
sshd_enable: YES -> YESYES
[test]:SYSRC -- END
```

3rd time:
```
[test]:SYSRC -- START
sshd_enable: YESYES -> YESYESYES
[test]:SYSRC -- END
```

Bastille just wants to make sure the service *really* starts :D

**Comment by chriswells0** [comment]
I tried to reproduce this and failed.

Running:
```
# bastille -v && freebsd-version -kru
0.6.20200414
12.1-RELEASE-p5
12.1-RELEASE-p5
12.1-RELEASE-p5
```

Perhaps fixed by 01eaccc1daf16ec56089d4274880ee09b7113c34?

**Comment by jaredwolff** [comment]
@chriswells0 confirmed. It appears too be fixed. I'll re-open if there's still issues.
